_summaries/hedman2021baking_1.md

```diff
@@ -1,15 +1,18 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{hedman2021baking}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author: timchen0618
+score: 8/10
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
+## What is the core idea?
+NeRF is a good method to recreate 3D scenes from viewpoints unobserved by the system. However, it is computationally intensive and cannot be applied in real-time. The authors proposed a novel approach to train NeRF called Sparse Neural Radiance Grid (SNeRG) and achieved real-time rendering. 
+
+## How is it realized (technically)?
+
```

Comment:
I think it will be better if you can write more about the technical details of this paper. For example, the architecture of networks used for training and rendering. The difference between rendering methods used in NeRF and SNeRG, and so on

Reply:
I think it is also nice to add how they minimize the storage and rendering time, and they also quantize all values in SNeSG representation.